test/integration/subnet_test.rb

```diff
@@ -7,6 +7,7 @@ class SubnetIntegrationTest < ActionDispatch::IntegrationTest
 
   test "create new page" do
     assert_new_button(subnets_path,"New Subnet",new_subnet_path)
+    assert page.has_link?('Parameters', :href => '#params')
```

Comment:
This just tests that there is a Parameters tab, would you like me to send an example on how to add an integration test  that checks you can add a parameter?